### PR TITLE
Changes made in Developer Guide

### PR DESCRIPTION
**Pull request by NishantSinghal19**
### Description
Made minor changes in the developer guide so new contributors will not get confused while setting up a repo.

### Issues Resolved
It is solving #1057 
### Check List
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: DEVELOPER_GUIDE.md [hunk 1]
```diff
@@ -16,10 +16,23 @@ We depend upon the version of node defined in [.nvmrc](.nvmrc).
 
 You will probably want to install a node version manager. [nvm](https://github.com/creationix/nvm) is recommended.
 
-To install and use the correct node version with `nvm`:
+To install and use the correct node version v with `nvm`:
 
+v - verion mentioned in [.nvmrc](.nvmrc).
+
+```
+nvm install v
```
Comment (BSFishy) [inline DEVELOPER_GUIDE.md:24]: You actually don't need to specify a version if you're in the same directory as the `.nvmrc` file. `nvm install` is sufficient enough

File: DEVELOPER_GUIDE.md [hunk 2]
```diff
@@ -16,10 +16,23 @@ We depend upon the version of node defined in [.nvmrc](.nvmrc).
...
+```
+
+And then also set the node version to use in your terminal
+
+```
+nvm use v
```
Comment (BSFishy) [inline DEVELOPER_GUIDE.md:30]: Same as previous comment

**Comment by NishantSinghal19** [comment]
I still mentioned v as to indicate the reader that they are using which version

**Review by BSFishy — APPROVED** [review]

**Review by ashwin-pc — APPROVED** [review]
Thanks! Hopefully no one gets confused here the next time :)

**Pull request merged**